content/user/apps/Route-coverage.md

```diff
@@ -14,7 +14,7 @@ Contrast observes the flow of data through routes in each of your applications.
 
 Contrast supports route coverage for the following frameworks: 
 
-* **Java:** Spring MVC 4.x 
+* **Java:** Jersey 2.x, Spring MVC 4.x 
```

Comment:
Tiny edit: "Jersey 2.x **and** Spring MVC 4.x"

Reply:
Go ahead and make whatever changes you deem the best